components/taupage.rst

```diff
@@ -602,12 +602,10 @@ Sample logging configuration::
          authlog_destination: scalyr
          scalyr_region: eu
          scalyr_account_key: "aws:kms:XYZABC..."
+         scalyr_agent_applog_sampling: '[{ match_expression: "INFO", sampling_rate: 0.1 }, { match_expression: "FINE", sampling_rate: 0 }]'
 
 In this example everything but the ``auth.log`` is logged to ``s3``, the ``auth.log`` is logged to ``Scalyr``. Logging is done with ``Fluentd``.
 
-.. NOTE::
-   If a logging section is present in ``senza.yaml`` then ``scalyr_account_key``, ``scalyr_application_log_parser``, ``scalyr_custom_log_parser`` and ``scalyr_region`` entries outside the scope of the logging section will be ignored.
-
```

Comment:
To refresh my memory: If one of these is set on the top level, but not in the `logging` section, the top level value will be taken. If they are set both on the top level and in the `logging` section, will value in the logging section will take precedence. Correct?

Reply:
Correct